onnx_neural_compressor/config.py

```diff
@@ -569,8 +626,70 @@ def register_supported_configs():
         config_cls.register_supported_configs()
 
 
+@dataclasses.dataclass
+class OperatorConfig:
+
+    def __init__(

```

Comment:
Since you are already using `dataclasses.dataclass`, it seems unnecessary to define `__init__`. I understand that you need to handle `weight_type` and `activation_type`. I would suggest utilizing `__post_init__` instead.